test/StaticValidatorTest.php

```diff
@@ -152,9 +152,38 @@ public function testPassingNullWhenSettingPluginManagerResetsPluginManager()
         $this->assertNotSame($plugins, StaticValidator::getPluginManager());
     }
 
-    public function testExecuteValidWithParameters()
+    public function parameterizedData()
     {
-        $this->assertTrue(StaticValidator::execute(5, 'Between', [1, 10]));
-        $this->assertTrue(StaticValidator::execute(5, 'Between', ['min' => 1, 'max' => 10]));
+        return [
+            'valid-positive-range'   => [5, 'Between', ['min' => 1, 'max' => 10], 'assertTrue'],
+            'valid-negative-range'   => [-5, 'Between', ['min' => -10, 'max' => -1], 'assertTrue'],
+            'invalid-positive-range' => [-5, 'Between', ['min' => 1, 'max' => 10], 'assertFalse'],
+            'invalid-negative-range' => [5, 'Between', ['min' => -10, 'max' => -1], 'assertFalse'],
+        ];
+    }
+
+    /**
+     * @dataProvider parameterizedData
+     */
+    public function testExecuteValidWithParameters($value, $validator, $options, $assertion)
+    {
+        $this->$assertion(StaticValidator::execute($value, $validator, $options));
```

Comment:
Lately I prefer asser the type first and then the value

asserInternalType(boolean, return) 
assertEquals(expected, return) 

The first ensure the type is a documented return type and also force exprcted to be the same type
The second one is the classic discrete comparison of values

This format has 2 goodnes:

* Keep clen the data provider from phpunit internals (method names) 
* Keep a consistent test format with a more clear error messages